csunplugged/topics/content/en/glossary/compress.md

```diff
@@ -0,0 +1,4 @@
+#Compress
+
+To compress something means to make it smaller.
+In computing we generally apply this to data compression, where we decrease the the number of bits we use to repesent a piece of information (e.g. decrease the size of a file) using a compression algorithm.
```

Comment:
It might just be me but I found the word 'this' in the following part confusing:
> In computing we generally apply this to data compression...
Because I thought it was referring to the action/process in the previous sentence, rather than the term. So it read something like:
> In computing we generally apply compression to data compression...

Reply:
I see what you mean, how about this simpler version instead:

> In computing, data compression means decreasing the number of bits...

I'll also remove the extra "the"

Reply:
Or maybe separate the definitions of "compress" and "data compression", or only have a definition of "data compression" and remove "compress".

Reply:
Replaced this with a definition of "compression":

> Compression is the process of making something something smaller.
Data compression is a core topic of Computer Science, and is the process of decreasing the number of bits we use to repesent a piece of information (e.g. decrease the size of a file) using a compression algorithm.

Reply:
That works really well

Reply:
Tim reviewed this and suggested a couple of changes, this is the current version:

> Compression is the process of trying to make something smaller.
On computers, data compression is about using a compression algorithm to reduce the size of files, so that they take up less space and can be transmitted faster.
This is generally done by trying to find repeated or predictable patterns in a file and replacing them with short codes that convey they same information.